pystiche_papers/li_wand_2016/_loss.py

```diff
@@ -165,6 +223,19 @@ def calculate_score(self, input_repr: torch.Tensor) -> torch.Tensor:
 def regularization(
     impl_params: bool = True, exponent: float = 2.0, score_weight: float = 1e-3,
 ) -> TotalVariationOperator:
+    r"""Regularization from :cite:`LW2016`.
+
+    Args:
+        impl_params: If ``True``, uses the parameters used in the reference
+            implementation of the original authors rather than what is described in
+            the paper. For details see below.
+        exponent: Parameter :math:`\beta` . A higher value leads to more smoothed
```

Comment:
Could you check if `\beta` is even used in the paper? I don't recall that.

Reply:
`\beta` is not used but nothing else is used either.
> For this reason, we penalize the squared gradient norm [20] to encourage smoothness in the synthesized image:

Reply:
In that case I would simply remove the first sentence.